walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -34,10 +34,12 @@
   // Some people that we can use in our tests.
   private static final String PERSON_A = "Person A";
   private static final String PERSON_B = "Person B";
+  private static final String PERSON_C = "Person C";
```

Comment:
I think I will keep the names as is because they are used as mandatory and optional in different tests - In this case the constants were already defined in the file so I decided to just reuse them

Reply:
sounds good, thank you!